CameraBrowser/CameraBrowserSample/CameraBrowserSample.entitlements

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>com.apple.security.device.camera</key>
+	<true/>
+	<key>com.apple.security.personal-information.photos-library</key>
+	<true/>
```

Comment:
I believe this should be in the `Entitlements.plist` file (as far as I can see `CameraBrowserSample.entitlements` is not used anywhere in the build?)